CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Release Notes
 
+### 0.11.0
+
+* Updates the Jackson dependency version to [2.9.10](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9#patches)
+to address three security vulnerabilities in Jackson:
```

Comment:
Oh, I was referring to the actual user doc (Readme) - we reference the jackson version in there I think.

Reply:
I didn't see one with a quick grep

Reply:
Ah, right, I was thinking of the dependency declaration part of the docs, but that's JJWT's .jar, not Jackson's.